Description: Do not run built programs during configure.
	When cross compiling the source, the host-specific programs
	built during the configuration phase are irrelevant for execution.
	Avoid running those executables. 

diff --git a/debian/patches/151-configure-crossbuild.diff b/debian/patches/151-configure-crossbuild.diff
new file mode 100644
index 0000000..e44d424
--- /dev/null
+++ b/debian/patches/151-configure-crossbuild.diff
@@ -0,0 +1,12 @@
+diff --git a/configure b/configure
+index ec30d77..42649b6 100755
+--- a/configure
++++ b/configure
+@@ -282,7 +282,6 @@ if (
+ else
+     if (
+           $CXX $CXXFLAGS -D__USE_BSD_SIGNAL __conftest.cc  -o __conftest || exit 1
+-          ./__conftest || exit 1
+        ) >/dev/null 2>&1; then
+         echo '-D__USE_BSD_SIGNAL'
+         CFLAGS="$CFLAGS -D__USE_BSD_SIGNAL"
diff --git a/debian/patches/series b/debian/patches/series
index 762c601..2c90b15 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -14,3 +14,4 @@
 140-telnetlogin_name_check.diff
 142-numeric_hosts.diff
 150-cross_configure.diff
+151-configure-cross.diff
-- 
2.15.1
